### ISSUE TITLE
Save video/still images

### ISSUE THREAD

**Issue by VeniceNerd** [moritzmhmk/homebridge-camera-rpi#18]
I just installed your disk image on my Pi and it works perfectly! I can see a love stream of my camera in HomeKit and I even get notifications with a still image from my Hue Motion Sensor placed in the same room!

I’m wondering if it is possible to somehow save detected motion to Dropbox in case I miss these notifications when they happen. 

Would it be possible to install MotionEyeOS (https://github.com/ccrisan/motioneyeos) on a raspberry Pi 3 and then install Homebridge with your plugin on that OS using the same camera? My thinking would be that MotionEye would take care of uploading detected motion videos to my dropbox while your plugin would allow HomeKit integration. 

Basically I love that I can now see live streams of my Pi in the Home app but I’m missing functionality to save videos to Dropbox to review events I missed at a later time.

**Comment by moritzmhmk** [comment]
It is not currently possible. Installing motion next to homebridge would not work since both will try to block the camera device.

**Comment by VeniceNerd** [comment]
Ah dang it. That’s currently the only shortcoming to this solution. I’m not sure if it’s within the realm of Homebridge but would love it some sort of Dropbox integration could be integrated into this plugin at some date to save videos when motion is detected.

**Comment by moritzmhmk** [comment]
see #26
